packages/dm-core-plugins/src/data-grid/DataGridPlugin.tsx

```diff
@@ -142,11 +142,16 @@ export function DataGridPlugin(props: IUIPlugin) {
               onClick={revertChanges}
               disabled={!isDirty}
               variant='outlined'
+              className='w-max h-max overflow-hidden'
```

Comment:
Does these buttons really need max height and width? And overflow-hidden? Seems strange...

Reply:
The buttons have an ::after effect that overflows the button container making a scrollbar appear that should not be there. I remember we looked at that together 😃

Reply:
w-max jsut means width: max-content, so the container gives as much space that the content needs to avoid wrapping of text etc.